guides/advanced_guides/faceted_search.md

```diff
@@ -0,0 +1,257 @@
+
+To get the facets distribution, you have to specify a list of facets for which to retrieve the count of matching documents using the `facetsDistribution` attribute.
+
+[Learn more about `facetsDistribution` in the search parameters](/guides/advanced_guides/search_parameters.md#the-facets-distribution)
+
+## Walkthrough
+
+Follow these steps to get started with faceting.
+
+Suppose that you manage a movie database. The first thing to do is to declare faceted attributes as follows:
+
+```bash
+$ curl \
+  -X POST 'http://localhost:7700/indexes/movies/settings' \
+  --data '{
+      "attributesForFaceting": [
+          "director",
+          "producer",
+          "genres",
+          "production_companies"
+      ]
+  }'
+```
+
+In the above example, `director`, `producer`, `genres` and `production_companies` will be used as facets.
+
+You can now search your documents and use query parameters.
+
+You can filter on facets. For instance, say you want to get movies matching "thriller" classified as either horror **or** mystery **and** directed by Jordan Peele. You have to use the following CURL command:
+
+```bash
+$ curl --get 'http://localhost:7700/indexes/movies/search' \
+    --data-urlencode 'q=thriller' \
+    --data-urlencode 'facetFilters=[["genres:Horror", "genres:Mystery"], "director:Jordan Peele"]'
+```
+
+You will get the following response:
+
+```json
+[
+  {
+    "id": 458723,
+    "title": "Us",
+    "director": "Jordan Peele",
+    "genres": [
+      "Thriller",
+      "Horror",
+      "Mystery"
+    ],
+    "overview": "Husband and wife Gabe and Adelaide Wilson take their kids to their beach house expecting to unplug and unwind with friends. But as night descends, their serenity turns to tension and chaos when some shocking visitors arrive uninvited.",
+    "production_companies": [
+      "Monkeypaw Productions"
+    ],
+...
+  },
+  {
+    "id": 419430,
+    "title": "Get Out",
+    "director": "Jordan Peele",
+    "genres": [
+      "Mystery",
+      "Thriller",
+      "Horror"
+    ],
+    "overview": "Chris and his girlfriend Rose go upstate to visit her parents for the weekend. At first, Chris reads the family's overly accommodating behavior as nervous attempts to deal with their daughter's interracial relationship, but as the weekend progresses, a series of increasingly disturbing discoveries lead him to a truth that he never could have imagined.",
+    "production_companies": [
+      "Monkeypaw Productions"
+    ],
+...
+  }
+],
+"offset": 0,
+"limit": 20,
+"nbHits": 2,
+"exhaustiveNbHits": false,
+"processingTimeMs": 4,
+"query": "thriller"
+```
+
+Now, if you want to know what the number of Batman movies per genre is, you have to use the following CURL command:
+
+```bash
+$ curl --get 'http://localhost:7700/indexes/movies/search' \
+    --data-urlencode 'q=Batman' \
+    --data-urlencode 'facetsDistribution=["genres"]'
+```
+
+You will get the following response:
+
+```json
+[
```

Comment:
```suggestion
"hits": [
```

Reply:
@bidoubiwa Would it be possible in a futur (futur) version of the CI to check code snippets anotatted as JSON for example? I know the `...` will not pass, but we will remove them.